include/nuttx/signal.h

```diff
@@ -44,6 +44,19 @@
   #define sigwork_init(work) (void)(work)
 #endif
 
+/* Internal signal set definitions */
+
+#define _NO_SIGNALS     ((uint32_t)0x00000000)
+#define _ALL_SIGNALS    ((uint32_t)0xffffffff)
+#define _SIGSET_NDX(s)  ((s) >> 5)    /* Get array index from signal number */
+#define _SIGSET_BIT(s)  ((s) & 0x1f)  /* Get bit number from signal number */
+#define _SIGNO2SET(s)   ((uint32_t)1 << _SIGSET_BIT(s))
```

Comment:
dup with:
https://github.com/apache/nuttx/pull/8885/files#diff-944d6d12b7cba69e3ea374135ebc541fd819308865868374776e7dc79d627195R57-R60

Reply:
Good catch.  That was a copy paste where I did not remove the source of the copy.

Done